roles/wildfly_prospero/tasks/main.yml

```diff
@@ -54,11 +57,33 @@
     group: "{{ wildfly_install.group }}"
     owner: "{{ wildfly_install.user }}"
     mode: 0750
+  become: yes
+  when:
+    - not eap_enable is defined or not eap_enable
+
+- name: Set install command arguments
+  block:
+    - name: Set default command arguments
+      ansible.builtin.set_fact:
+        prospero_command_args: ''
+    - name: Override repositories parameter
+      ansible.builtin.set_fact:
+        prospero_command_args: "{{ prospero_command_args }} --repositories {{ eap_prospero_override_repos }}"
+      when:
+        - eap_prospero_override_repos is defined
+    - name: Override provisioning manifest
+      ansible.builtin.set_fact:
+        prospero_command_args: "{{ prospero_command_args }} --manifest {{ wildfly_install_manifest_dir }}/wildfly-{{ wildfly_version }}-manifest.yaml"
+      when:
+        - not eap_enable is defined or not eap_enable
+    - name: Display prospero command arguments
```

Comment:
@spyrkob I would keep the debug stmt but add a 

```
when: 
  - wildfly_prospero_debug_enabled is defined and wildfly_prospero_debug_enabled
```

Reply:
Done, thanks!